src/config.rs

```diff
@@ -86,7 +86,10 @@ where F: FnMut(&str, &str) -> Result<(), String> {
                 let warn = |msg: &str| eprintln!("{}:{}: {}", path.to_string_lossy(), i + 1, msg);
                 match (line.chars().next(), line.split_once('=')) {
                     (Some('#' | ';') | None, _) => (), // Comment or empty line
-                    (_, Some((k, v))) => kv_fn(k.trim_end(), v.trim()).unwrap_or_else(|r| warn(&format!("{k}: {r}"))),
+                    (_, Some((k, v))) => {
```

Comment:
There seem to be a couple formatting changes without code change, but which increases line count. Do you mind reverting those?